pkg/porter/create.go

```diff
@@ -4,40 +4,66 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"get.porter.sh/porter/pkg"
 	"get.porter.sh/porter/pkg/config"
 )
 
-func (p *Porter) Create() error {
-	fmt.Fprintln(p.Out, "creating porter configuration in the current directory")
+// Create function creates a porter configuration in the specified directory or in the current directory if no directory is specified.
+func (p *Porter) Create(bundleName string) error {
+	// Normalize the bundleName by removing trailing slashes
+	bundleName = strings.TrimSuffix(bundleName, "/")
 
-	err := p.CopyTemplate(p.Templates.GetManifest, config.Name)
+	// Use the current directory if no directory is passed
+	if bundleName == "" {
+		bundleName = p.FileSystem.Getwd()
+	}
+
+	// Check if the directory in which bundle needs to be created already exists.
+	// If not return error.
+	// It takes both the relative path and absolute path into consideration.
+	// For example if we want to create a bundle named mybundle in an existing directory /home/user we can call porter create /home/user/mybundle or porter create mybundle in the /home/user directory.
+	// If we are in a directory /home/user and we want to create mybundle in the directory /home/user/directory given the directory exists, 
+	// we can call porter create directory/mybundle from the /home/user directory or with any relative paths' combinations that one can come up with.
```

Comment:
To clarify, when we do this `porter create scripts/bundles/porter` is the name going to be this as well?  I feel like we may have to split on the `/` to `os.Stat` the directory path if it doesn't/does exist but only use the last name from the `/` when creating the bundle. This seems like it may be doing the former.

Reply:
No the name will be let say if you create /home/sbdtu5498/gentoo/nix/os. let say both gentoo and nix exist then it will work and will create bundle in the directory named os. If os doesn't exist, it will create a directory called os. But if nix doesn't exist, then it will throw error no such directory exist. This is the exact behavior that helm commands show. I tried to closely replicate it. Hope it makes sense. I will add few more descriptor comments.

Reply:
// Only condition to use porter create with absolute and relative paths is that all the directories in the path except the last one should strictly exist.

Reply:
I will add this line here. Now it will make sense.